src/netius/test/common/http2.py

```
@@ -0,0 +1,353 @@
+    netius.common.GOAWAY, payload=struct.pack("!II", 3, 0x00) + b"bye"
+)
+
+WINDOW_UPDATE_FRAME = _pack_frame(
+    netius.common.WINDOW_UPDATE, payload=struct.pack("!I", 4096)
+)
+
+
+class HTTP2ParserTest(unittest.TestCase):
+
+    def setUp(self):
+        self.settings = dict(netius.common.HTTP2_SETTINGS_OPTIMAL)
```

Pylint (4.0.5)

[error] Module 'netius.base.common' has no 'HTTP2_SETTINGS_OPTIMAL' member

(E1101)